gateway-js/src/__tests__/loadSupergraphSdlFromStorage.test.ts

```diff
@@ -23,402 +30,69 @@ describe('loadSupergraphSdlFromStorage', () => {
     const result = await loadSupergraphSdlFromStorage({
       graphRef,
       apiKey,
-      endpoint: mockCloudConfigUrl,
+      endpoint: mockCloudConfigUrl1,
+      errorReportingEndpoint: mockOutOfBandReporterUrl,
       fetcher,
       compositionId: null,
+    });
+    expect(result).toMatchInlineSnapshot(sampleSchema);
+  });
 
+  it('Queries alternate Uplink URL if first one fails', async () => {
+    mockSupergraphSdlRequest('originalId-1234').reply(500);
+    mockOutOfBandReportRequestSuccess();
+    mockSupergraphSdlRequestIfAfter('originalId-1234', mockCloudConfigUrl2).reply(
```

Comment:
This looks a lot like what `mockSupergraphSdlRequestSuccessIfAfter` does for us, can we adapt that function to also accept a URL?

Reply:
I added a URL parameter to `mockSupergraphSdlRequest()` and set it here for clarity.